schemaregistry/cache/lrucache.go

```diff
@@ -131,6 +131,19 @@ func (c *LRUCache) Delete(key interface{}) {
 	}
 }
 
+// Clear clears the cache
+func (c *LRUCache) Clear() {
+	c.cacheLock.Lock()
+	for key, value := range c.lruElements {
```

Comment:
Is there a more efficient way to clear than iterating through all the elements?

Reply:
I believe this is idomatic go before go 1.21, where they added a `clear()` method.  In any case, these maps are not expected to get that large as they store one entry per subject